web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -602,6 +602,8 @@ $.extend(fixmystreet.set_up, {
         }
         if (data.category_extra) {
             $category_meta.replaceWith( data.category_extra );
+            document.dispatchEvent(new Event('category_extra_loaded'));
```

Comment:
At the end of this function is a `report_new:category_change` (jQuery) event being triggered, which you could use instead of a new one?